### ISSUE TITLE
Add support for FTP transfers

### ISSUE THREAD

**Issue by mauromsl** [openlibhums/lockss#2]
When an article links to supporting material that is not hosted on Janeway, the (C)LOCKSS crawler is sometimes unable to preserve the content behind systems that do not explicitely allow for a permission statement to be posted in order to harvest web content (e.g videos hosted on Vimeo).

The team at CLOCKSS has proposed that we could instead send them the files and metadata over FTP in these cases. This could be an opt-in feature that would behave similarly to the portico plugin (https://github.com/BirkbeckCTP/portico)

The CLOCKSS file transfer guidelines are available here: https://lockss.github.io/clockss-file-transfer-guidelines

**Comment by ajrbyers** [comment]
We could add support for this to our FTP transporter idea.

**Comment by mauromsl** [comment]
> We could add support for this to our FTP transporter idea.

Yep, now that we have two different use cases there is value on making that abstraction
